pkg/sql/distsql_physical_planner.go

```diff
@@ -4490,7 +4521,9 @@ func (dsp *DistSQLPlanner) NewPlanningCtx(
 	txn *kv.Txn,
 	distributionType DistributionType,
 ) *PlanningCtx {
-	return dsp.NewPlanningCtxWithOracle(ctx, evalCtx, planner, txn, distributionType, physicalplan.DefaultReplicaChooser)
+	return dsp.NewPlanningCtxWithOracle(
+		ctx, evalCtx, planner, txn, distributionType, physicalplan.DefaultReplicaChooser, roachpb.Locality{},
```

Comment:
ditto, these empty localities seem odd. We sure we don't need filtering here?

Reply:
`NewPlanningCtx` is doing the same thing it always did: return a planning context that will plan on any node, using the default replica oracle and all. It calls `NewPlanningCtxWithOracle` to do so which is the more powerful, takes-more-options, but is more annoying to call version. It passes it an empty filter since it doesn't setup filtered planning.